### PR TITLE
Add .rst file documentation for the draw_legend function and fix two misspellings in spatial.py

### PR DESCRIPTION
**Pull request by hdumke**
@lwasser Added documentation for draw_legend functon to spatial-raster.rst and fixed misspelling of the word 'output' in stack_raster_tifs code in spatial.py.  While reviewing the code for the draw_legend function in spatial.py, I also noticed that the word 'unique' was misspelled.  I did not see a reported issue for that elsewhere so I went ahead and fixed that as well. These updates are for the items listed in this issue: https://github.com/earthlab/earthpy/issues/63

### CODE REVIEW

File: docs/spatial-raster.rst [hunk 1]
```diff
@@ -49,3 +49,57 @@ Example:
 
     # Stack landsat tif files
     arr, arr_meta = es.stack_raster_tifs(all__paths, destfile)
+
+
+Draw Legend
```
Comment (lwasser) [inline docs/spatial-raster.rst]: Very nice submission @hdumke !! One small change. While this does render just fine, normally it is requested that rst headings are the same line length. So I am requesting you remove a few of the tildas `~` like i did below. Once you do that I will approve and merge your pr! 

```
Draw Legend
~~~~~~~~~~~
```

Comment (hdumke) [reply docs/spatial-raster.rst]: @lwasser made the requested change to remove the extraneous tildas so that they only underline the heading.

**Comment by lwasser** [comment]
This looks good @hdumke thank you! i will merge this later today once i see it builds properly!

**Comment by lwasser** [comment]
well oddly this still says travis is building but i can assure you all tests have passed. [see here](https://travis-ci.org/earthlab/earthpy/builds/448914525?utm_source=github_status&utm_medium=notification). I am merging this. thank you again @hdumke !!

**Pull request merged**